content/blog/esc-versioning-launch/index.md

```diff
@@ -0,0 +1,102 @@
+6. Create a new environment and import that environment revision using that tag
+
+<span style="color: red;">**Update Video**</span>
+{{< video title="Pulumi ESC Versioning Demo" src="" autoplay="true" loop="true" >}}
+
+## Real-world Scenarios
+Here are a few examples of how you might use these features:
+
+- **Global Infrastructure Update**: You need to deploy a new configuration to a global service spanning multiple regions. Tag the updated configuration as `Next` and deploy it to one region for testing. After verifying stability, you can confidently roll it out to other regions. If issues arise, quickly revert to the previous `Stable` version, ensuring resilient and minimally disruptive deployments.
+- **Platform Team Releases**: If you're a Platform team responsible for generating configurations, by tagging releases with labels like `v1.0-stable` and `v1.2-development`, you can clearly communicate which configurations are production-ready, keeping other teams informed about the ongoing development, and fostering seamless collaboration.
+- **Code Review and Debugging**: You're working in a development environment where the latest revision includes multiple recent changes, and you notice that the application is experiencing performance issues. By using the Compare Revisions Side-by-Side feature, you can quickly identify the specific changes that might be causing the problem, enabling faster troubleshooting and resolution.
+- **A/B Testing**: You can try A/B testing a new checkout flow (checkout-B) against your existing one (checkout-A). Using version tags, you can gradually roll out each to a subset of users, monitoring conversion rates. If checkout-B wins, seamlessly switch all users over with a simple environment reference update – no downtime, no disruption.
+
+These examples are just a glimpse of how Pulumi ESC’s versioning capabilities can streamline your development and deployment process. The possibilities are vast, and you can tailor your tagging strategy to fit your specific needs and operational workflows.
+
+## What’s coming in the future
+
+We're just getting started with versioning in Pulumi ESC! We have exciting plans to further enhance this feature, making it even more powerful. Here's a sneak peek at what's on the horizon:
+
+- **Contextual Comments**: Add comments to versions providing valuable context for your team.
+- **Granular Version Policies**: Enforce organizational best practices and compliance requirements by defining policies that govern how versions are created and managed.
+  
+## Conclusion
+
+The versioning capabilities in Pulumi ESC mark a significant step towards a more developer-friendly and efficient platform for configuration and secrets management. These features provide a robust foundation for secure, reliable, and auditable deployments, whether you're rolling out infrastructure updates, conducting A/B testing, or ensuring compliance with industry regulations.
```

Comment:
See above, suggest we remove a/b testing

Reply:
done